src/ImageCard.js

```diff
@@ -1,57 +1,63 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+const MAX = 5
+
 class ImageCard extends React.Component {
   constructor(props) {
     super(props)
 
     this.state = {
-      showResults: false
+      isHidden: this.props.content ? this.props.content.length >= MAX : false
     }
     this.onClick = this.onClick.bind(this)
   }
  
-  onClick(){
+  onClick() {
     this.setState({
-      showResults: !this.state.showResults
+      isHidden: !this.state.isHidden
     })
   }
 
   render(){
-    let {iconSrc, iconDescription, content} = this.props
+    const {iconSrc, iconDescription, content} = this.props
+
+    let visibleContent = (max, content) => {
+      return content.slice(0, max).map((item) => {
+        return item.url ?
+                  <li className={`url`} style={{marginBottom: `0.3rem`}} key={item.text}><a href={item.url} key={item.text}>{item.text}</a></li> :
+                  <li className={`text`} style={{marginBottom: `0.3rem`}} key={item.text}>{item.text}</li>
+                })
```

Comment:
Align closing curly brace and parenthesis (like you did in `ExtendableSpeciesCard`).